### PR TITLE
#720386 and #720633: Improve developer-visible interface of module

### PR DESCRIPTION
**Pull request by reinouts**
Including new Java actions that should be used from now on, instead of microflows.

### CODE REVIEW

**Comment by dlhartveld** [comment]
Great, this solves at least the second point in issue #4.

File: README.md [hunk 1]
```diff
@@ -23,15 +23,20 @@ In general, a Mendix Push Notifications solution consists of two parts: the `Pus
 - Android 4.0 and newer
 - iOS 7.0 and newer
 
-## Limitations
+## Limitations and known issues
 
-This project assumes that the mobile app and the "back-end" part will be co-located in the same application.
+### This project assumes that the mobile app and the "back-end" part will be co-located in the same application.
```
Comment (sjoerdbreur) [inline README.md]: Need to make these headers a bit smaller

File: README.md [hunk 2]
```diff
@@ -171,17 +188,14 @@ Fill in the title and the message in the form and press `Send immediate` button.
...
-|SendMessageImmediately |A microflow to send a message to GCM/APNs immediately          |
-|SendMessageQueued      |A microflow to send a message to GCM/APNs using a queue        |
-|SendMessagesImmediately|A microflow to send multiple messages to GCM/APNs immediately  |
-|SendMessagesQueued     |A microflow to send multiple messages to GCM/APNs using a queue|
+|SendMessage            |An action to send a message to GCM/APNs immediately            |
+|QueueMessage           |An action to send a message to GCM/APNs using a queue          |
 
-> Note: for sending queued messages to work, you need to ensure that the scheduled event `PushQueue` is active. In a Free App environment this does not require an extra step, but for TAP environments you will need to explicitly activate the scheduled event in the Mendix Cloud Portal.
+> Note: for sending queued messages to work, you need to ensure that the scheduled event `PushQueue` is active. For TAP environments you will need to explicitly activate the scheduled event in the Mendix Cloud Portal, for Free Apps the scheduled events feature is not supported.
```
Comment (sjoerdbreur) [inline README.md]: Mention explicitly that queueing message thus doesn't work for free apps

File: README.md [hunk 3]
```diff
@@ -141,6 +156,8 @@ You can proceed by repackaging the project into a zip file and using PhoneGap Bu
 
 For more information about PhoneGap Build, you can refer to their [documentation](http://docs.build.phonegap.com/en_US/index.html).
 
+> Note that a free PhoneGap Build account is limited to a single application, whereas paid plans support multiple applications.
```
Comment (sjoerdbreur) [inline README.md]: Clarify a bit more what this implies (eg, need to delete existing app to create new one)

**Pull request merged**